swagger_ui/core.py

```diff
@@ -142,12 +141,13 @@ def get_config(self, host):
                 config = _load_config(config_file.read())
         elif self.config_spec:
             config = _load_config(self.config_spec)
+        else:
+            raise RuntimeError('No config found!')
 
         version = config.get('openapi', '2.0.0')
         if Version(version) >= Version('3.0.0'):
             for server in config.get('servers', []):
-                server['url'] = re.sub(r'//[a-z0-9\-\.:]+/?',
-                                       '//{}/'.format(host), server['url'])
+                server['url'] = server['url']
```

Comment:
I can't understand this, A = A?

Reply:
Good point, that block is actually obsolete without the transformation I will remove it.